custom-routes-vis/src/main/webapp/script.js

```diff
@@ -99,7 +100,7 @@ function updateCoordinates(lat, lng, containerId) {
 }
 
 /**
- * Hides 'place marker' button and waits for user to select coordinate for
+ * Hides 'place marker' button and waits for user to select coordinates for
```

Comment:
You are not only hiding the 'place marker' button, right?

Reply:
I am only hiding the place marker in the new code version